### DISCUSSION TITLE
'use_zoe_depth'

### DISCUSSION THREAD

**Discussion by ExecuteAI** [deforum-art/deforum-stable-diffusion#208] [Q&A]
Does anybody know what to do about this? 

WARNING: Setting 'use_zoe_depth' has been renamed to 'depth_algorithm' with value 'Midas+AdaBins (old)'. The saved settings file will reflect the change

WARNING: Value '1' for setting 'cn_1_weight' has been replaced with '0:(1)'. The saved settings file will reflect the change
WARNING: Value '0' for setting 'cn_1_guidance_start' has been replaced with '0:(0)'. The saved settings file will reflect the change
WARNING: Value '1' for setting 'cn_1_guidance_end' has been replaced with '0:(1)'. The saved settings file will reflect the change
WARNING: Value '1' for setting 'cn_2_weight' has been replaced with '0:(1)'. The saved settings file will reflect the change
WARNING: Value '0' for setting 'cn_2_guidance_start' has been replaced with '0:(0)'. The saved settings file will reflect the change
WARNING: Value '1' for setting 'cn_2_guidance_end' has been replaced with '0:(1)'. The saved settings file will reflect the change
WARNING: Value '1' for setting 'cn_3_weight' has been replaced with '0:(1)'. The saved settings file will reflect the change
WARNING: Value '0' for setting 'cn_3_guidance_start' has been replaced with '0:(0)'. The saved settings file will reflect the change
WARNING: Value '1' for setting 'cn_3_guidance_end' has been replaced with '0:(1)'. The saved settings file will reflect the change
WARNING: Value '1' for setting 'cn_4_weight' has been replaced with '0:(1)'. The saved settings file will reflect the change
WARNING: Value '0' for setting 'cn_4_guidance_start' has been replaced with '0:(0)'. The saved settings file will reflect the change
WARNING: Value '1' for setting 'cn_4_guidance_end' has been replaced with '0:(1)'. The saved settings file will reflect the change
WARNING: Value '1' for setting 'cn_5_weight' has been replaced with '0:(1)'. The saved settings file will reflect the change
WARNING: Value '0' for setting 'cn_5_guidance_start' has been replaced with '0:(0)'. The saved settings file will reflect the change
WARNING: Value '1' for setting 'cn_5_guidance_end' has been replaced with '0:(1)'. The saved settings file will reflect the change

**Comment by hithereai** [comment]
Wrong repository, and you don't need to do anything - it's just telling you that the value name has been changed. It will run 1:1 as you're used to.
